apps/jfr-native-image-performance/quarkus-demo.jfc

```diff
@@ -0,0 +1,1007 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The only real difference is that this file specifies the commit threshold for `jdk.ThreadPark` to be very low (so we can spam that event during the test). I think I'm also shortening the sampling interval for the execution sampler as well. Since it samples randomly, it should populate the method and stacktrace repos more. I suppose I could generate the config on the fly too, but decided to commit the file for convenience. I can look into changing this after I resolve your other suggestions.

Reply:
The problem with committing the file is that over time it gets outdated, plus there's no easy way to see what is different from the default.

Reply:
That's a good point. I have pushed an update using `jfr config` to generate the JFC file at runtime.